test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -654,17 +657,19 @@ private void BuildProjectJson(string projectDirectory)
 
             var result = new BuildPJCommand()
                 .WithCapturedOutput()
+                .WithForwardingToConsole()
                 .Execute(projectFile);
 
             result.Should().Pass();
         }
 
         private void MigrateProject(params string[] migrateArgs)
         {
-            var result =
-                MigrateCommand.Run(migrateArgs);
-
-            result.Should().Be(0);
+            new TestCommand("dotnet")
+                    .WithForwardingToConsole()
+                    .Execute($"migrate {string.Join(" ", migrateArgs)}")
+                    .Should()
```

Comment:
If you want to avoid long lines, `Should` often needs its own line, e.g. when followed by an assertion method with a longer name, so I tend to go more vertical for the sake of consistent formatting. It's just a matter of taste though.